thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detection/alert/DetectionAlertTaskRunner.java

```diff
@@ -45,15 +93,19 @@ public DetectionAlertTaskRunner() {
 
     try {
       long alertId = ((DetectionAlertTaskInfo) taskInfo).getDetectionAlertConfigId();
+      DetectionAlertConfigDTO alertConfig = loadDetectionAlertConfig(alertId);
 
-      DetectionAlertFilter alertFilter = detectionAlertTaskFactory.loadAlertFilter(alertId, System.currentTimeMillis());
+      DetectionAlertFilter alertFilter = detAlertTaskFactory.loadAlertFilter(alertConfig, System.currentTimeMillis());
       DetectionAlertFilterResult result = alertFilter.run();
 
-      Set<DetectionAlertScheme> alertSchemes = detectionAlertTaskFactory.loadAlertSchemes(alertId, taskContext, result);
+      this.currentAndBaselineLoader.fillInCurrentAndBaselineValue(result.getAllAnomalies());
```

Comment:
In a new design, we decided to move filling current & baseline value back to the detection pipeline again. So currentAndBaselineLoader won't be necessary anymore. let's leave it for now.

Reply:
I will probably keep it here for now as it is outside the scope of this PR. We can remove it in a separate PR.

Reply:
Ok. Please leave a TODO here with context